app/frontend/package-lock.json

```diff
@@ -16290,11 +16290,6 @@
       "resolved": "https://registry.npmjs.org/vue/-/vue-2.6.10.tgz",
       "integrity": "sha512-ImThpeNU9HbdZL3utgMCq0oiMzAkt1mcgy3/E6zWC/G6AaQoeuFdsl9nDhTDU3X1R6FK7nsIUuRACVcjI+A2GQ=="
     },
-    "vue-analytics": {
```

Comment:
Side effects from other tickets?

Reply:
yeah... as this ticket was resolving errors during container building, and this remnant in the package-lock did provoke a warning, i thought it acceptable to include the cleanup in this ticket.